soccer/gameplay/single_robot_sequence.py

```diff
@@ -0,0 +1,51 @@
+import behavior_sequence
+import single_robot_behavior
+import role_assignment
+
+#A sequence to be carried out by a single robot, attempting to add non-single robot behaviors will raise exceptons
+class SingleRobotSequence(behavior_sequence.BehaviorSequence, single_robot_behavior.SingleRobotBehavior):
```

Comment:
If you subclass from SingleRobotCompositeBehavior instead of SingleRobotBehavior, you can inherit assign_roles() for free and avoid having to copy that code here.